Yasgui/packages/utils/src/index.ts

```diff
@@ -159,3 +159,8 @@ export enum EXPLAIN_PLAN_TYPE {
   EXPLAIN = "explain",
   CHAT_GPT_EXPLAIN = "gpt",
 }
+
+export interface RanRequestCountInfo {
+  queryCount: number;
```

Comment:
`queriesCount`, `updatesCount` because we can have more than one query or update running. Right?